python/ray/train/v2/_internal/execution/controller/controller.py

```diff
@@ -393,11 +392,15 @@ async def _step(self) -> TrainControllerLoopIterationResult:
                     next_state=FinishedState(),
                 )
             if worker_group_status.errors:
+                worker_group_error = WorkerGroupError(
+                    error_message=worker_group_status.get_error_string(),
+                    worker_failures=worker_group_status.errors,
+                )
```

Comment:
can we make a `WorkerGroupError` constructor on the `WorkerGroupPollStatus`?

Ex: `worker_group_status.get_error() -> WorkerGroupError`